tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualAppInfo.scala

```diff
@@ -351,8 +351,7 @@ object QualAppInfo extends Logging {
       if (angleBracketsCount == 0 && parenthesesCount == 0 && char.equals(',')) {
         individualSchema += tempStringBuilder.toString
         tempStringBuilder.setLength(0)
-      }
-      else {
+      } else {
         tempStringBuilder.append(char);
       }
     }
```

Comment:
down below in the split 

> x.split(":", 2)(1).split("\\.\\.\\.")(0)
>     val completeTypes = completeSchema.map(x => x.split(":", 2)(1))

we should check to make sure there are 2 elements afte rthe split before blindingly accessing (1)